include/roboteam_ai/control/positionControl/pathTracking/PidTracking.h

```diff
@@ -0,0 +1,51 @@
+//
+// Created by ratoone on 24-01-20.
+//
+
+#ifndef RTT_PIDTRACKING_H
+#define RTT_PIDTRACKING_H
+
+#include "roboteam_utils/Position.h"
+#include "utilities/Constants.h"
+#include "roboteam_utils/Vector2.h"
+#include "interface/api/Output.h"
+#include "roboteam_utils/pid.h"
+#include "control/positionControl/PositionControlUtils.h"
+
+namespace rtt::ai::control{
+
+/**
+ * Path tracking algorithm. See method computePath for details.
+ */
+class PidTracking {
+private:
+    static constexpr double MAX_VELOCITY = Constants::MAX_VEL();
+
+    // the PID controllers on the two axes
+    PID xPid = PID(0,0,0,0);
+    PID yPid = PID(0,0,0,0);
```

Comment:
confident this is just default construction but it's ok :)

Reply:
There is no default constructor for PID